test/controllers/organizations/invitations_controller_test.rb

```diff
@@ -30,7 +67,7 @@ class Organizations::InvitationControllerTest < ActionDispatch::IntegrationTest
             first_name: "John",
             last_name: "Doe",
             email: "john@example.com",
-            roles: "admin"
+            roles: "staff"
```

Comment:
I just changed this because I just happened to see we used "admin" as a param in another test for this controller, so I figured it is kind of nice to have both values used in separate tests. This is not an error.